src/mmw/js/src/draw/views.js

```diff
@@ -19,13 +19,30 @@ var ToolbarView = Marionette.LayoutView.extend({
     template: toolbarTmpl,
 
     regions: {
-        selectAreaRegion: '#select-area-region',
+        selectTypeRegion: '#select-area-region',
         drawAreaRegion: '#draw-area-region',
         placeMarkerRegion: '#place-marker-region'
     },
 
+    modelEvents: {
+        'change:toolsEnabled': 'manageOutlineLayer'
+    },
+
+    initialize: function() {
+        var map = App.getLeafletMap();
+        var ofg = this.model.outlineFeatureGroup = L.featureGroup();
```

Comment:
Yeah, it would be good to consolidate them.  Since `leafletMap` is accessible to anyone, it makes sense that other modules would be interested in turning on/off layer display.